app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
 
     <application
         android:name=".MyApplication"
+        android:largeHeap="true"
```

Comment:
@the-dagger The Glide library we are using here should actually take care of these things right? Even in developer page you mentioned it is recommended to use Glide library for most purposes. And for the caching part, I guess we are doing that using the Glide library here.
```
.diskCacheStrategy(DiskCacheStrategy.SOURCE)
```